scripts/module/gen_doc.sh

```diff
@@ -16,8 +16,10 @@ fi
 module=$(echo ${MODULE} | cut -d'_' -f 2)
 tf="$(./scripts/stack/gen_module.sh ${TARGET})"
 detectors=""
+metrics=""
 if compgen -G "${TARGET}/detectors-*.tf" > /dev/null; then 
     detectors="$(sed -n 's/^.*name.*=.*detector_name_prefix.*"\(.*\)")$/* \1/p' ${TARGET}/detectors-*.tf | sort -fdbiu)"
+    metrics=$(grep "data('" ${TARGET}/detectors-*.tf | sed -E "s/^.*data\('([^']*).*$/\1/g" | sort -fdbiu | uniq | sed ':a;N;$!ba;s/\n/,/g')
```

Comment:
A one or two lines comment above this line to explain the goal would be fine since it's hardly understandable.

Reply:
ok